craft_cli/dispatcher.py

```diff
@@ -101,6 +104,7 @@ class BaseCommand:
     """
 
     common = False
+    hidden = False
```

Comment:
As `common` and `hidden` are mutually exclusive would an enum be a good replacement for the two boolean values?

Reply:
I like "the rule of three". Two cases are too few to start generalizing.